src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -321,37 +318,13 @@ private void InformAboutFilteredOutProjects()
 
         internal /* for testing purposes */ static Project[] GetProjectsForRulesetBinding(bool isFirstBinding,
             Project[] allSupportedProjects,
-            IUnboundProjectFinder unboundProjectFinder,
             ILogger logger,
             IThreadHandling threadHandling)
         {
             threadHandling.ThrowIfOnUIThread();
 
-            // If we are already bound we don't need to update/create rulesets in projects
-            // that already have the ruleset information configured
-            var projectsToUpdate = allSupportedProjects;
-
-            if (isFirstBinding)
-            {
-                logger.WriteLine(Strings.Bind_Ruleset_InitialBinding);
-            }
-            else
-            {
-                var unboundProjects = unboundProjectFinder.GetUnboundProjects()?.ToArray() ?? new Project[] { };
-                projectsToUpdate = projectsToUpdate.Intersect(unboundProjects).ToArray();
-                var upToDateProjects = allSupportedProjects.Except(unboundProjects);
-                if (upToDateProjects.Any())
-                {
-                    logger.WriteLine(Strings.Bind_Ruleset_SomeProjectsDoNotNeedToBeUpdated);
-                    var projectList = string.Join(", ", upToDateProjects.Select(p => p.Name));
-                    logger.WriteLine($"  {projectList}");
-                }
-                else
-                {
-                    logger.WriteLine(Strings.Bind_Ruleset_AllProjectsNeedToBeUpdated);
-                }
-            }
-            return projectsToUpdate;
+            // We no longer use rulesets => no projects need ruleset binding
+            return Array.Empty<Project>();
```

Comment:
`BindingProcessImpl` is a complicated class. We might remove it entirely in the near future.
For now, I've settled for making the minimum changes to keep it compiling and the tests passing.